workspace/apps/pidp/src/app/features/shell/pages/support-error/support-error.page.html

```diff
@@ -0,0 +1,10 @@
+<ui-dashboard-header
+  [theme]="headerConfig.theme"
+  [allowMobileToggle]="headerConfig.allowMobileToggle"></ui-dashboard-header>
+<div class="error-page">
+  <h1>An error has occurred</h1>
```

Comment:
I think there is more to the ticket in the comments.  The support page is for when an error occurs and we need to provide contact information, otherwise if it was generic like this we could move it into the RootRouteModule.

Reply:
lol I totally missed the comment section